vscode/src/chat/chat-view/SimpleChatPanelProvider.ts

```diff
@@ -1035,21 +1035,23 @@ export class SimpleChatPanelProvider implements vscode.Disposable, ChatSession {
         const authStatus = this.authProvider.getAuthStatus()
 
         // Count code generated from response
-        const hasCode = countGeneratedCode(messageText.toString())
-        const responeEventName = hasCode?.charCount ? 'hasCode' : 'noCode'
+        const generatedCode = countGeneratedCode(messageText.toString())
+        const responseEventName = generatedCode.charCount > 0 ? 'hasCode' : 'noCode'
```

Comment:
Minor nit - maybe we should name this in a more V2-oriented manner, i.e. 

```suggestion
        const responseEventAction = generatedCode.charCount > 0 ? 'hasCode' : 'noCode'
```

Reply:
pushed this change 🚀